internal/humio/license.go

```diff
@@ -0,0 +1,31 @@
+package humio
+
+import (
+	"fmt"
+
+	jose "github.com/go-jose/go-jose/v4"
```

Comment:
`jose` alias is not needed here since we're importing `/jwt` as a separate import